lua/telescope/builtin.lua

```diff
@@ -774,4 +774,33 @@ builtin.man_pages = function(opts)
   }):find()
 end
 
+builtin.colorscheme = function(opts)
+  opts = opts or {}
+
+  local colors = vim.fn.getcompletion('', 'color')
+
+  pickers.new(opts,{
+    prompt = 'Change Colorscheme',
+    finder = finders.new_table {
+      results = colors
+    },
+    previewer = previewers.man.new(opts),
```

Comment:
I don't think this should set a `previewer`.

I have an idea for how to preview the colorscheme -- but it will take a quite bit of time on my side.